packages/spatial/src/camera/components/CameraComponent.ts

```diff
@@ -28,13 +28,14 @@ import { ArrayCamera, PerspectiveCamera } from 'three'
 import { defineComponent } from '@etherealengine/ecs/src/ComponentFunctions'
 import { addObjectToGroup, removeObjectFromGroup } from '../../renderer/components/GroupComponent'
 
+export const cameraNear = 0.1
```

Comment:
This is not a const, it is a default value that is configurable. Why is this being exported?

Reply:
It is being exported so it can be used to calculate the avatar radius in spawnAvatarReceptor.ts

Reply:
removed the const and put the default value back in the array declaration 
spawnAvatarReceptor now updates the avatar radius by fetching the camera's near value from the camera component

Reply:
Updated the avatar Radius to be calculated as part of the avatar collider's update